### PR TITLE
Create 2DArray_DS.java

### PR DESCRIPTION
**Pull request by anahita-singla**
**Fixes issue:**

https://www.hackerrank.com/challenges/2d-array/problem

**Changes:**

Provided solution for the above problem on Hackerrank.

### CODE REVIEW

File: code/online_challenges/src/hackerrank/2DArray-DS/2DArray_DS.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import java.io.*;
+import java.math.*;
+import java.security.*;
+import java.text.*;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.regex.*;
+
+public class Solution {
```
Comment (arnavb) [inline code/online_challenges/src/hackerrank/2DArray-DS/2DArray_DS.java]: Folder name should be all lowercase as well and separated by underscores. In addition, reindent this file with 4 spaces. When making these changes, do not create a new pull request, just push to this same branch when updating your code.

Comment (anahita-singla) [reply code/online_challenges/src/hackerrank/2DArray-DS/2DArray_DS.java]: Other than indenting it and selecting 4 spaces indentation in github editor, do I have to do something else? It'll be of great help if you tell me.

Comment (anahita-singla) [reply code/online_challenges/src/hackerrank/2DArray-DS/2DArray_DS.java]: Also I'm trying to apply 4 space indentation but it keeps on going to 8 tabs even after committing.

Comment (arnavb) [reply code/online_challenges/src/hackerrank/2DArray-DS/2DArray_DS.java]: @anahita-singla use an online or local Java formatter that can use 4 spaces. I recommend the one in IntelliJ.

**Pull request merged**